apps/asap-server/src/handlers/invalidate-cache/index-handler.ts

```diff
@@ -0,0 +1,24 @@
+import { S3Event } from 'aws-lambda';
+import {
+  CloudFrontClient,
+  CreateInvalidationCommand,
+} from '@aws-sdk/client-cloudfront';
+import { cloudfrontDistributionId } from '../../config';
+import logger from '../../utils/logger';
+
+export const createHandler =
+  (distributionId: string) =>
+  async (event: S3Event): Promise<void> => {
+    logger.debug(`event: ${JSON.stringify(event)}`);
+    const client = new CloudFrontClient({ apiVersion: '2020-05-31' });
+    const command = new CreateInvalidationCommand({
```

Comment:
why do you use CreateInvalidationCommand and not `CreateInvalidation` ?https://docs.aws.amazon.com/AWSJavaScriptSDK/latest/AWS/CloudFront.html#createInvalidation-property